src/components/EarningsTable/EarningsTable.styles.js

```diff
@@ -0,0 +1,185 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+/**
+ * @constant
+ * @type {object}
+ * @description possible colors for earning table component
+ * @see {@link https://colors.artyclick.com/color-name-finder/}  - to learn how
+ *  color names are generated
+ */
+const COLORS = {
+  appleGreen: '#86C232',
+  lavenderPinocchio: '#E0E0E0',
+  carbonGrey: '#585B5D',
+  black: '#000',
+  white: '#fff',
+  feta: '#F3F9EB',
+};
+
+/**
+ * @constant
+ * @type {object}
+ * @name EarningsTableTopBarStyles
+ * @description styles for EarningsTableTopBar component
+ */
+const earningsTableTopBarStyles = {
+  csvLink: {
+    color: COLORS.appleGreen,
+    display: 'flex',
+    alignItems: 'flex-end',
+    textDecoration: 'none',
+  },
+  earningsTableTopBar: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    padding: '40px 0px 20px 0px',
+  },
+  filterAvatar: {
+    backgroundColor: COLORS.feta,
+    color: COLORS.appleGreen,
+    marginLeft: '0.75rem',
+    width: '25px',
+    height: '25px',
+  },
+
+  iconFilter: {
+    color: COLORS.appleGreen,
+  },
+
+  filterButtonText: {
+    color: COLORS.black,
+    fontSize: '1.2em',
+  },
+  dateFiterButonSmallText: {
+    color: COLORS.carbonGrey,
+    fontSize: '0.8em',
+    textAlign: 'left',
+  },
+  dateFiterButonMediumText: {
+    fontSize: '1.1em',
+    textAlign: 'left',
+  },
+  actionButton: {
+    color: COLORS.appleGreen,
+    cursor: 'pointer',
+  },
+  actionButtonIcon: {
+    position: 'relative',
+    top: '2px',
+  },
+  topBarActions: {
+    width: '60%',
+  },
+
+  earningsTableTopTitle: {
+    padding: '0px 0px 0px 10px',
```

Comment:
Where possible, we should be using `theme.spacing()` to specify padding and margins.
`theme` is available to `makeStyles` as the first parameter passed to the function.

Reply:
Noted @nmcharlton